packages/flutter/lib/src/widgets/overlay.dart

```diff
@@ -115,6 +120,21 @@ class OverlayEntry {
     _overlay._didChangeEntryOpacity();
   }
 
+  /// Whether this entry is a semantics barrier for the entries below it.
+  ///
+  /// If `true`, entries below this entry will not be included in the semantic
+  /// tree. That means they are not reachable via accessibility means.
```

Comment:
same comment here. "This means, for example, that they are not reachable using accessibility tools."